mitmproxy/tools/web/webaddons.py

```diff
@@ -46,7 +46,7 @@ def open_browser(url: str) -> bool:
     browsers = (
         "windows-default", "macosx",
         "wslview %s",
-        "x-www-browser %s", "gnome-open %s",
+        "x-www-browser %s", "gnome-open %s", "xdg-open",
```

Comment:
Is this related to the Docker change? I don't think this even works when mitmproxy is running inside a container...

Reply:
This isn't related to the Docker change but was actually part of the previous PR I made (#4435), which I believe was merged, so I don't know why this shows up as a commit since both my fork and the main branch are on par

Reply:
https://github.com/mitmproxy/mitmproxy/blob/4bd7b6c4eadeaca712f63e0e73f20bcf6aadbffb/mitmproxy/tools/web/webaddons.py#L49
The main branch has this commit too. 
Apologies for any confusion

Reply:
Please rebase this PR to have a clean commit history and minimize the diff.